taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/LaundryStateDto.java

```diff
@@ -21,9 +21,13 @@
     private Integer version;
 
     public LaundryStateDto(LaundryStateEntity laundryStateEntity) {
-        this.laundryId = laundryStateEntity.getId();
```

Comment:
null might be passed

Reply:
so it should be reworked in all constructors like dto-entity/entity-dto, right?